packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -996,3 +996,46 @@ class PlatformViewSurface extends LeafRenderObjectWidget {
       ..updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// Integrates an Android view with Flutter's compositor, touch, and semantics subsystems.
+///
+/// The compositor integration is done by adding a [PlatformViewLayer] to the layer tree. [PlatformViewLayer]
+/// isn't supported on all platforms. Custom Flutter embedders can support
+/// [PlatformViewLayer]s by implementing a SystemCompositor.
+///
+/// The widget fills all available space, the parent of this object must provide bounded layout
+/// constraints.
+///
+/// If the associated platform view is not created, the [AndroidViewSurface] does not paint any contents.
+///
+/// See also:
+///
+///  * [AndroidView] which embeds an Android platform view in the widget hierarchy using a [TextureLayer].
+///  * [UIKitView] which embeds an iOS platform view in the widget hierarchy.
+class AndroidViewSurface extends PlatformViewSurface {
```

Comment:
@blasten This can be it's own class or we can add `pointTransformer` field to the `PlatformViewController` class. Then Android could just use `PlatformViewSurface`. What are your thoughts on this?

Reply:
gotcha. nice adapter for these two interfaces :)